src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,83 @@
 import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  selectedTodoId?: number;
+  setSelectedTodo?: (value: Todo) => void;
```

Comment:
why do you make `setSelectedTodo` and `selectedTodoId`  not required props?

Reply:
because selected Todo is possibly null, I've changed that and added additional checks..